application/views/admin/globalsettings/_security.php

```diff
@@ -147,23 +147,29 @@
                         <?php eT("IP whitelist:"); ?>
                     </label>
                     <textarea class="form-control" id='loginIpWhitelist' name='loginIpWhitelist'><?php echo htmlspecialchars((string) Yii::app()->getConfig('loginIpWhitelist')); ?></textarea>
-                    <span class='hint'><?php eT("List of IP addresses to exclude from the maximum login attempts check. Separate each IP address with a comma or a new line."); ?></span>
+                    <div class='form-text'><?php eT("List of IP addresses to exclude from the maximum login attempts check. Separate each IP address with a comma or a new line."); ?></div>
                 </div>
 
                 <div class="mb-3">
                     <label class="form-label" for='maxLoginAttempt'>
                         <?php eT("Maximum number of attempts:"); ?>
                     </label>
                     <div class="">
-                        <input class="form-control" type="number" min="0" name="maxLoginAttempt" value="<?= Yii::app()->getConfig('maxLoginAttempt') ?>" />
+                        <input class="form-control" type="number" min="1" step="1" pattern="^\d*$" name="maxLoginAttempt" placeholder="<?= gT("Disabled") ?>"
+                            value="<?= App()->getConfig('maxLoginAttempt') !== "" ?? intval(App()->getConfig('maxLoginAttempt')) ?>"
+                        />
+                        <div class="form-text"><?= gT("Set an empty value disable brute force protection. Number of attemps are deleted each time.") ?></div>
                     </div>
                 </div>
                 <div class="mb-3">
                     <label class="form-label" for='timeOutTime'>
                         <?php eT("Lockout time in seconds (after maximum number of attempts):"); ?>
                     </label>
                     <div class="">
-                        <input class="form-control" type="number" min="0" name="timeOutTime" value="<?= Yii::app()->getConfig('timeOutTime') ?>" />
+                        <input class="form-control" type="number" min="0" step="1" pattern="^\d*$" name="timeOutTime" placeholder="<?= gT("Disabled") ?>"
+                            value="<?= App()->getConfig('timeOutTime') !== "" ?? intval(App()->getConfig('timeOutTime')) ?>"
+                        />
+                        <div class="form-text"><?= gT("Set an empty value or 0 disable brute force protection. Number of attemps are deleted each time.") ?></div>
```

Comment:
Please write this sentence like this:
"Set an empty value or 0 to disable brute force protection. Number of attempts are deleted each time."